src/NHSD.BuyingCatalogue.Identity.Api/Settings/ApiResourceSettingCollection.cs

```diff
@@ -15,7 +16,7 @@ public sealed class ResourceSetting
 
         public ApiResource ToResource()
         {
-            return new ApiResource(ResourceName, DisplayName);
+            return new ApiResource(ResourceName, DisplayName, new List<string>(){ ApplicationClaimTypes.Organisation });
```

Comment:
This needs to be configurable. During development we only hardcoded it to see if we could get it working. Not all API resources will want the Organisation scope. This will involve a change to the appsettings.json (& the development version).